packages/office-ui-fabric-react/src/components/Rating/Rating.test.tsx

```diff
@@ -78,6 +78,78 @@ describe('Rating', () => {
     expect((ratingInputs[4] as HTMLInputElement).checked).to.be.eq(true);
   });
 
+  it('Deciamal Rating not aggregate', () => {
```

Comment:
This looks like it was an issue when checked in. If you could adjust the language for these, that would help with consistency.